tasks/copy.js

```diff
@@ -19,14 +19,34 @@ module.exports = {
     return src("src/**/**/*.scss").pipe(dest("dist/scss"));
   },
 
-  // Copy material icons to /dist/img/material-icons
-  copyIcons() {
-    dutil.logMessage("copyIcons", "Copying Material icons to dist/img/material-icons");
-    return src(["node_modules/@material-design-icons/svg/outlined/*"])
-      .pipe(dest("dist/img/material-icons"));
+  // Copy material icons to packages/usa-icon/dist
+  copyMaterialIcons() {
+    dutil.logMessage("copyMaterialIcons", "Copying Material icons to packages/usa-icon/dist/img/material-icons");
+    return src(["node_modules/@material-design-icons/svg/filled/*"])
+      .pipe(dest("packages/usa-icon/dist/img/material-icons"));
   },
 
-  // Copy images to /dist directory
+  // Copy USWDS package icons to packages/usa-icon/dist
+  copyPackageIcons() {
+    dutil.logMessage("copyPackageIcons", "Copying USWDS Icons to packages/usa-icon/dist/img");
+    return src([
+      "packages/**/src/img/**/*{png,svg,gif}",
+      // exclude hero images and favicons
```

Comment:
I imagined these exclusions are only temporary until we start distributing these files inside their own packages.

Reply:
Wouldn't we want both? 

1. Package for single use
2. Dist for supporting current users

This task might fit better in its respective package.

Reply:
Makes sense to me! Thanks for the suggestion. Moved the scripts to usa-icon in [05439c6](https://github.com/uswds/uswds/pull/4878/commits/05439c6ee10ed47fff7c260b326a984c0eeab132)